cmp/form/FormModel.ts

```diff
@@ -274,4 +308,83 @@ export class FormModel extends HoistModel {
             }
         );
     }
+
+    private initPersist({
+        fieldsToInclude = null,
+        fieldsToExclude = null,
+        path = 'formValues',
+        ...rootPersistWith
+    }: FormPersistOptions) {
+        const fieldNameMap = this.createFormFieldsToPersistListing(
```

Comment:
Might just be me mixing Java with JS, but to me a Map suggests an object with key/value pairs. I might just call this "fieldsToPersist" or "fieldNamesToPersist".